docs/tutorial/intro_to_zydeco.md

```diff
@@ -0,0 +1,168 @@
+# Intro to Zydeco
+## Types
+Zydeco has two main types: value type (`A`) and computation type (`B`). 
+
+Value type can be: 
+- `X`: type variable defined using `data`.
+- `Int | String | Boolean | Unit`: basic primitive type with built-in functions.
+- `Thunk(B)`: suspend a computation type and consider it as a value type. 
+
+Computation type can be:
+- `X`: type variable defined using `codata`.
+- `Ret(A)`: one of computation types we can run, the other is `OS`.
```

Comment:
`Ret(A)` classifies computations that return `A` values and `OS` classifies computations that can be run as a process that interacts with the `OS`.